Recruit-Training/Embedded-Recruit-Training/Core/Src/GreenLedToggleTask.c

```diff
@@ -1,5 +1,8 @@
 #include "GreenLedToggleTask.h"
 
+static const uint32_t GREEN_LED_STATUS_STDID = 0xDDD;
+static const uint32_t GREEN_LED_TOGGLE_FREQ = 10000;
```

Comment:
This period is incorrect, it would lead to a frequency of 0.1Hz instead of 10Hz.